### PR TITLE
[WIP] - GraphQL

### PR DESCRIPTION
**Pull request by jaypathar**

### CODE REVIEW

File: _GraphQL/index.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    type Todo {
+        id : ID!,
+        title : String!,
+        completed : Boolean
+    }
+    type Query {
+        getTodos : [Todo]
+        getAllUsers : [Users]
+    }
+    `,
```
Comment (RenishIs) [inline _GraphQL/index.js]: Let's Extract Types into diffrent file

**Review by RenishIs — CHANGES_REQUESTED** [review]

**Review by RenishIs — APPROVED** [review]

File: _GraphQL/index.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+//   app.use("/", expressMiddleware(server));
+
+//   // listening on port 5000.
+//   app.listen(5000, () => {
+//     console.log("Listening on port 5000....");
+//   });
+// }
+
+// // function call.
+// startServer();
```
Comment (albiorixvimal) [inline _GraphQL/index.js]: Create a separate file if this code required otherwise remove it

File: _GraphQL/index.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+  const app = express();
+  // defining GraphQL schema.
+  const server = new ApolloServer({
+    // modularizing GraphQL schema.
+    typeDefs: [Users, Todo],
+    // defining resolvers for the schema.
+    resolvers: {
+      Query: {
+        getTodos: todosResolver,
+        getAllUsers: userResolver,
```
Comment (albiorixvimal) [inline _GraphQL/index.js]: rename it to `getUsers` to consistency in naming conventions

**Review by albiorixvimal — APPROVED** [review]
Please address minor comments and merge it

**Pull request merged**